src/main.rs

```diff
@@ -289,8 +296,11 @@ fn record(
             }
         }
         // Sleep until the next expected sample time
-        let sleep_time = sample_time.get_sleep_time();
-        std::thread::sleep(std::time::Duration::new(0, sleep_time));
+        //
+        match sample_time.get_sleep_time(){
+            Ok(sleep_time) => {std::thread::sleep(std::time::Duration::new(0, sleep_time));},
+            Err(behind_time) => {eprintln!("Behind expected sample time by {} nanoseconds, results may be inaccurate. Try sampling at a lower rate", behind_time);},
```

Comment:
The one thing I might find useful here is if this told me how to sample! What is the current rate, and how to I set a lower rate?